web/src/main/java/se/inera/intyg/intygstjanst/web/integration/validator/SendMessageToCareValidator.java

```diff
@@ -76,23 +74,33 @@ public enum ErrorCode {
     @Autowired
     private ArendeRepository messageRepository;
 
+    @Autowired
+    private CSSendMessageToCareValidator csSendMessageToCareValidator;
+
     public List<String> validateSendMessageToCare(SendMessageToCareType sendMessageToCareType) {
-        List<String> validationErrors = new ArrayList<>();
-        String personnummeer = sendMessageToCareType.getPatientPersonId().getExtension();
+        final var validationErrors = new ArrayList<String>();
+        final var certificateId = sendMessageToCareType.getIntygsId().getExtension();
+        final var personnummer = sendMessageToCareType.getPatientPersonId().getExtension();
 
         validateSkickatAv(sendMessageToCareType.getSkickatAv().getPart().getCode(), validationErrors);
-        validateMeddelandeId(sendMessageToCareType.getMeddelandeId(), validationErrors);
         validateMessageSubject(sendMessageToCareType.getAmne().getCode(), validationErrors);
-        validateThatCertificateExists(sendMessageToCareType.getIntygsId().getExtension(), personnummeer, validationErrors);
         validateConsistencyForQuestionVsAnswer(sendMessageToCareType, validationErrors);
+        validateConsistencyForKomplettering(sendMessageToCareType, validationErrors);
+        validateMeddelandeId(sendMessageToCareType.getMeddelandeId(), validationErrors);
         validatePaminnelse(sendMessageToCareType, validationErrors);
         validateConsistencyOfSubject(sendMessageToCareType, validationErrors);
-        validateConsistencyForKomplettering(sendMessageToCareType, validationErrors);
-        validateTestCertificate(sendMessageToCareType.getIntygsId().getExtension(), validationErrors);
 
+        if (certificateExists(certificateId)) {
+            validateThatCertificateExists(certificateId, personnummer, validationErrors);
+            validateTestCertificate(certificateId, validationErrors);
+            return validationErrors;
+        }
+
+        csSendMessageToCareValidator.validate(certificateId, personnummer, validationErrors);
```

Comment:
Realised now that we need to make it feature toggled whether it should call CS or not.  

Suggest that we add a spring profile, similar to webcert, where we can control if CS should be called or not. If no profile it should not call CS.